plugins/base/src/main/kotlin/renderers/html/innerTemplating/DefaultTemplateModelFactory.kt

```diff
@@ -91,6 +93,30 @@ public class DefaultTemplateModelFactory(
             ?: DokkaBaseConfiguration.defaultFooterMessage)
     )
 
+    private fun calculateSourceUrlFromSourceLinks(): String? {
+        val githubLinkRegex = Regex("http(s)?://github\\.com/([\\w,\\-_]+)/([\\w,\\-_]+)/.*")
+
+        fun parseGithubInfo(link: String): Pair<String, String>? {
+            val (
+                _, // entire match
+                _, // optional 's' in http
```

Comment:
What do you think about ```https?://github\\.com/([\\w,\\-_]+)/([\\w,\\-_]+)/.*``` where `s` is not a capturing group?

Reply:
yeah, agree, that should work, will fix it, thx!